src/shared/components/person/profile.tsx

```diff
@@ -125,6 +129,15 @@ export class Profile extends Component<any, ProfileState> {
       this.state.personRes.person_view.person.id
     );
   }
+  isBlocked() {
```

Comment:
Use a `get`, then you don't have to call `isBlocked()`, you can just use `isBlocked`